packages/fast-components/src/dialog/dialog.template.ts

```diff
@@ -0,0 +1,34 @@
+import { html } from "@microsoft/fast-element";
+import { ref, when } from "@microsoft/fast-element/dist/directives/";
```

Comment:
Do we need to export these from the root of this package?

Reply:
Rob was doing some investigation on that. I believe we've updated 'when' (I did that yesterday) so that can come in.

Reply:
resolving this for `when` - unsure when/if ref will get exported.